sky/packages/sky/lib/src/gestures/arena.dart

```diff
@@ -45,6 +45,7 @@ class GestureArenaEntry {
 class _GestureArenaState {
   final List<GestureArenaMember> members = new List<GestureArenaMember>();
   bool isOpen = true;
+  bool isHeld = false;
```

Comment:
Closing it doesn't release it.  Closing prevents new members from joining.  I think @krisgiesing is right that the enum representation is going to have four states, which means we might as well use two bools.